apps/academy-wep/components/learn/learning-module-card.tsx

```
@@ -1,180 +1,191 @@
-              <span
-                key={index}
-                className="text-xs bg-gray-100 text-gray-600 px-2 py-1 rounded-full"
-              >
-                {tag}
-              </span>
-            ))}
-          </div>
+					<div className="flex flex-wrap items-center gap-2 mb-4 min-h-[28px]">
+						{module.tags.map((tag, index) => (
+							<span
+								key={index}
```

Biome (1.9.4)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)